blogposts/2020/3.18/combo-post.md

```diff
@@ -0,0 +1,39 @@
+---
+title: 'Small but mighty new features'
+author: 'Loïc Guychard, Adam Harvey, Eric Fritz'
+publishDate: 2020-07-20T00:00-07:00
+tags: ["blog"]
+slug: small-and-mighty-features
+heroImage: https://about.sourcegraph.com/sourcegraph-mark.png
+published: true
+---
+## Improved add / update repository flow for users trying private versions of Sourcegraph (#[11044](https://github.com/sourcegraph/sourcegraph/issues/11044))
+
+Developer: [Loïc Guychard](https://github.com/lguychard)
+
+When you’re getting started setting up a Sourcegraph instance, it can be confusing to start the process of adding a code host. You may not be sure that Sourcegraph
+will protect the privacy of your data because there is no communication about how the software will interact with your repositories. We’ve added a new language in
+Sourcegraph 3.18 that clarifies what information we access within the Sourcegraph UI so you can move forward with confidence.
+
+![Add repository flow](./images/add-repository-flow.png "Privacy feedback in Sourcegraph UI")
+
+## Campaigns now support GitLab (#[11586](https://github.com/sourcegraph/sourcegraph/issues/11586))
+Developer: [Adam Harvey](https://github.com/LawnGnome)
+
+You asked, we implemented. We’ve received a lot of feedback from developers using GitLab as their code hosts that campaign capabilities would be useful, so we added
+them in the Sourcegraph 3.18 release. This makes campaigns available to more organizations and helps make Sourcegraph more universal. See Adam’s introductory video below:
+
+[![Campaigns for Gitlab](https://img.youtube.com/vi/KatiVJ4D3H4/maxresdefault.jpg)](https://youtu.be/KatiVJ4D3H4)
```

Comment:
You should use the following embed code for adding videos (you will need to update the URLs below.

```
<p class="container">
  <div style="padding:56.25% 0 0 0;position:relative;">
    <iframe src="https://www.youtube.com/embed/{VIDEO_ID}" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
  </div>
  <p style="text-align: center"><a href="https://youtu.be/{VIDEO_ID}" target="_blank">View on YouTube</a></p>
</p>
```

Reply:
You can find this by looking at the blog post template in the handbook in raw format: https://raw.githubusercontent.com/sourcegraph/about/master/handbook/product/product_management/release_blog_post_template.md